src/ast.ts

```diff
@@ -526,26 +525,7 @@ function getFieldEntryKey(node: FieldNode): string {
   return node.alias ? node.alias.value : node.name.value;
 }
 
-/**
- * Resolves the field on the given source object. In particular, this
- * figures out the value that the field returns by calling its resolve function,
- * then calls completeValue to complete promises, serialize scalars, or execute
- * the sub-selection-set for objects.
- */
-export function resolveFieldDef(
-  compilationContext: CompilationContext,
-  parentType: GraphQLObjectType,
-  fieldNodes: FieldNode[]
-): Maybe<GraphQLField<any, any>> {
-  const fieldNode = fieldNodes[0];
-
-  if (versionInfo.major < 16) {
-    const fieldName = fieldNode.name.value;
-    return getFieldDef(compilationContext.schema, parentType, fieldName as any);
-  }
-
-  return getFieldDef(compilationContext.schema, parentType, fieldNode as any);
-}
+export { resolveFieldDef };
```

Comment:
Decided to leave this file as an entry point for `resolveFieldDef`. @boopathi wdyt?